docs/source/how-to/deploying-substra/upgrade-notes.rst

```diff
@@ -4,6 +4,38 @@
 Upgrade notes
 *************
 
+.. _ops upgrade notes 0.38:
+
+Substra 0.38.0
+--------------
+
+This version upgrades the integrated PostgreSQL databases chart. After this uprade, you may get the following warning:
+
+.. note::
+
+   You need to apply the following command for the `substra-backend` and the `orchestrator`
+
+.. code-block::
+
+   WARNING: database "<db_name>" has a collation version mismatch                                                                                                                                                     │
+   DETAIL:  The database was created using collation version 2.31, but the operating system provides version 2.36.                                                                                                        
+   HINT:  Rebuild all objects in this database that use the default collation and run ALTER DATABASE <db_name> REFRESH COLLATION VERSION, or build PostgreSQL with the right library version.
+
+#. Gather required info:
+
+   * Kubernetes namespace (here ``$NAMESPACE``)
+   * Helm release (hereafter ``$POD_NAME``, can be obtained from ``kubectl get pods -n $NAMESPACE``)
+
+#. Run the following commands
+   #. For the backend:
```

Comment:
I think this is not intented?

<img width="714" alt="image" src="https://github.com/Substra/substra-documentation/assets/135698225/7ec81964-8d76-4559-9f92-1e29da34b70d">

---

```suggestion
#. Run the following commands

   #. For the backend:
```